### ISSUE TITLE
Hitting a null object error sporadically

### ISSUE THREAD

**Issue by JoeSzymanskiFAN** [sorenlouv/backport#497]
We have started making use of the backport CLI tool to do some backport automation with our repo and occasionally we are hitting an issue I can't quite track down. When we hit this issue, we are getting the following error:

```
{"status":"failure","commits":[],"error":{},"errorMessage":"Cannot read properties of null (reading '\''remoteConfigHistory'\'')"}
```

The redacted command we are running is:
```
backport --repo [redacted] \
         --accessToken [redacted] \
         --pullNumber 7979 \
         --noFork \
         --nonInteractive \
         --commitConflicts \
         --label "automated backport" \
         --source-branch "example-source-branch" \
         --target-branch "example-target-branch" \
         --dryRun
```

Most of the time, we see everything working, but sometimes this problem gets hit for a specific PR, and I haven't been able to get it working at all for that PR.

Any direction to investigate would be helpful.

**Comment by sorenlouv** [comment]
Can you check if the logs contain the stack trace for the error `Cannot read properties of null (reading 'remoteConfigHistory')`?

Logs are located in `~/.backport/`

**Comment by sorenlouv** [comment]
Thanks. The problem is happening here so it looks like there is no `mergeCommit`. Has the PR in question been merged?

https://github.com/sorenlouv/backport/blob/24dc0f87e68e2ca7986e138d363e04908d80e201/src/lib/sourceCommit/parseSourceCommit.ts#L281

**Comment by JoeSzymanskiFAN** [comment]
Yes, it has been merged via a squash into the source branch in question.

**Comment by JoeSzymanskiFAN** [comment]
Just want to confirm that I pulled the latest version directly and was able to finish a dry run against the PRs that previously were failing. I think I'll need this to be released at some point officially.

**Comment by sorenlouv** [comment]
Thanks for testing it, much appreciated! I've released the fix in [v9.4.6](https://github.com/sorenlouv/backport/commit/b58dec2372a07ec9a8d72da68154ba71cac7c151)
